sdk/servicebus/microsoft-azure-servicebus/src/main/java/com/microsoft/azure/servicebus/primitives/ClientEntity.java

```diff
@@ -107,15 +107,10 @@ protected final void throwIfClosed(Throwable cause) {
     }
 
     @Override
-    protected void finalize() {
-        try {
-            if (!this.isClosed) {
-                this.close();
-            }
-
-            super.finalize();
-        } catch (Throwable e) {
-            //Ignore
-        }
+    protected void finalize() throws Throwable {
```

Comment:
>throws Throwable  [](start = 30, length = 17)

what happens when we throw from finalize method?

Reply:
It is just ignored.